docs/spec/WIP_provisioning/overview.md

```diff
@@ -0,0 +1,229 @@
+# Fee Distribution
+
+## Overview
+
+Fees are pooled separately and withdrawn lazily, at any time.  They are not
+bonded, and can be paid in multiple tokens.  An adjustment factor is maintained
+for each validator and delegator to determine the true proportion of fees in
+the pool they are entitled too.  Adjustment factors are updated every time a
+validator or delegator's voting power changes.  Validators and delegators must
+withdraw all fees they are entitled too before they can bond or unbond Atoms.
```

Comment:
"entitled *to*", explain why they must withdraw all fees (in order to enable the lazy calculation)

Reply:
WIP, next PR